app/DynamicsAdapter/DynamicsAdapter.Web/Mapping/MappingProfile.cs

```diff
@@ -28,13 +27,13 @@ public SearchRequestAutoMapperProfile()
                  .ForMember(dest => dest.FirstName, opt => opt.MapFrom(src => src.PersonGivenName))
                  .ForMember(dest => dest.LastName, opt => opt.MapFrom(src => src.PersonSurname))
                  .ForMember(dest => dest.DateOfBirth, opt => opt.MapFrom(src => src.PersonBirthDate))
-                 .ForMember(dest => dest.Identifiers, opt => opt.MapFrom(src => src.Identifiers));       
+                 .ForMember(dest => dest.Identifiers, opt => opt.MapFrom(src => src.Identifiers));
 
             CreateMap<PersonSearchAccepted, SSG_SearchApiEvent>()
               .ForMember(dest => dest.Id, opt => opt.MapFrom(src => Guid.NewGuid()))
               .ForMember(dest => dest.ProviderName, opt => opt.MapFrom(src => src.ProviderProfile.Name))
               .ForMember(dest => dest.TimeStamp, opt => opt.MapFrom(src => src.TimeStamp))
-              .ForMember(dest => dest.EventType, opt => opt.MapFrom(src =>"Accepted"))
+              .ForMember(dest => dest.EventType, opt => opt.MapFrom(src => "Accepted"))
```

Comment:
Can we move to keys.cs like `EVENT_ACCEPTED = "Accepted"`

Reply:
ok